src/chrome/content/rules/MPort.ua.xml

```diff
@@ -2,17 +2,18 @@
 	For other bigmir coverage, see Bigmirnet.xml.
 
 -->
-<ruleset name="MPort.ua" default_off="refused">
+<ruleset name="MPort.ua">
 
 	<target host="mport.ua" />
 	<target host="www.mport.ua" />
 
 
-	<!--	Not secured by server:
-					-->
-	<!--securecookie host="^\.ua$" name="^bm_lang$" /-->
+	<securecookie host=".+" name=".+" />
 
 
+	<rule from="^http://www\.mport\.ua/"
+		to="https://mport.ua/" />
```

Comment:
Please add/update the comments for such redirects.

Reply:
@J0WI oops, fixed.